src/App.tsx

```diff
@@ -1,24 +1,197 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
+import { TodoCreate } from './components/TodoCreate';
+import { TodoInfo } from './components/TodoInfo';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
 
-const USER_ID = 0;
+const USER_ID = '6757';
+
+enum SortType {
+  completed,
+  active,
+  all,
+}
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>();
+  const [selectedForm, setSelectedForm] = useState(SortType.all);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [countComplited, setCountComplited] = useState(false);
+  const [countNotComplited, setCountNotComplited] = useState(false);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const setBlock = {
+    setCountComplited,
+    setCountNotComplited,
+  };
+
+  const statusComplited = {
+    countComplited,
+    countNotComplited,
+    todosFromServer,
+  };
+
+  const fetchTodos = (url: string) => {
+    client
+      .get(url)
+      .then((todos) => {
+        setTodosFromServer(todos as Todo[]);
+      })
+      .catch(() => setErrorMessage('Unable to update a todo'));
+  };
+
+  const askTodos = debounce((url) => fetchTodos(url), 1000);
+
+  const clearCompleted = async (status: string) => {
+    if (todosFromServer) {
+      todosFromServer.forEach((todo => {
+        switch (status) {
+          case 'completed':
+            if (todo.completed) {
+              client
+                .delete(`/todos/${todo.id}`)
+                .finally(() => {
+                  askTodos('/todos?userId=6757');
```

Comment:
please use the constant
```suggestion
                  askTodos(`/todos?userId=${USER_ID}`);
```